metricbeat/module/couchdb/_meta/test/serverstats.json

```diff
@@ -0,0 +1,43 @@
+  "couchdb": {
```

Comment:
Is this an incomplete JSON? It should start by `{`. The error in travis seems caused by this.

Reply:
I may have overlooked this field. I made the correction. I think the error in travis isn't about this. CouchDB returns an error because database does not exist in Docker. This causes the TestFetch() methods in integration tests to fail. I'm trying to fix this error, but I haven't found a solution yet.